lib/gringotts/gateways/stripe.ex

```diff
@@ -1,9 +1,9 @@
 defmodule Gringotts.Gateways.Stripe do
-  
+
```

Comment:
@jyotigautam  you have changed `stripe.ex` unnecessarily, there will be no breaking changes this time, but before committing code you should check that you are not adding any other unnecessary files.